content/kubermatic/main/installation/install-kkp-CE/add-seed-cluster-CE/_index.en.md

```diff
@@ -171,42 +318,215 @@ metadata:
+aws-eu-central-1a:
+  country: DE
+  location: EU (Frankfurt)
+  spec:
+    aws:
+      region: eu-central-1
+# Datacenter for AWS 'eu-west-1' region
+aws-eu-west-1a:
+  country: IE
+  location: EU (Ireland)
+  spec:
+    aws:
+      region: eu-west-1
+```
+{{% /tab %}}
+{{% tab name="Azure" %}}
+```yaml
+# Datacenter for Azure 'westeurope' location
+azure-westeurope:
+  country: NL
+  location: West Europe
+  spec:
+    azure:
+      location: westeurope
+```
+{{% /tab %}}
+{{% tab name="GCP" %}}
+```yaml
+# Datacenter for GCP 'europe-west3' region
+# this is configured to use three availability zones and spread cluster resources across them
+gce-eu-west-3:
+  country: DE
+  location: Frankfurt
+  spec:
+    gcp:
+      region: europe-west3
+      regional: true
+      zoneSuffixes: [a,b,c]
+```
+{{% /tab %}}
+{{% tab name="vSphere" %}}
+```yaml
+# Datacenter for a vSphere setup available under https://vsphere.hamburg.example.com
+vsphere-hamburg:
+  country: DE
+  location: vSphere Hamburg
+  spec:
+    vsphere:
+      cluster: Hamburg
+      datacenter: Hamburg
+      datastore: hamburg1
+      endpoint: "https://vsphere.hamburg.example.com"
+      rootPath: /Hamburg/vm/kubernetes
+      templates:
+        ubuntu: ubuntu-20.04-server-cloudimg-amd64
+```
+{{% /tab %}}
+{{% tab name="Other Providers" %}}
+For additional providers supported by KKP, please check out our [DatacenterSpec CRD documentation]({{< ref "../../../references/crds/#datacenterspec" >}})
+for the respective provider you want to use.
+{{% /tab %}}
+{{< /tabs >}}
+
+Note that for many private datacenter providers (such as OpenStack, vSphere or Nutanix), the `templates` section is mandatory
+for providing default images to use for various OSes.
+
+### Configure Backup Locations
+
+Within your `Seed` resource, the `.spec.etcdBackupRestore` key configures cluster backup locations. Depending
+on which storage backend you chose to work with before, this will look slightly different. Below you will find
+two examples. Be aware that you can configure multiple destinations and as such could configure both the MinIO
+backup location and another S3-compatible storage backend at the same time.
+
+Omit `defaultDestination` if you do not wish to enable default etcd backups on all clusters.
+Additional backup locations can also be added after installation either by updating the `Seed` resource or via
+[the UI]({{< ref "../../../tutorials-howtos/administration/admin-panel/backup-buckets/" >}}).
+
+#### MinIO Backup Location
+
+If MinIO was [installed from the provided Helm chart](#configure-cluster-backups), the etcd backup location configuration
+should look like this:
+
+```yaml
+# snippet, not a valid seed resource!
+apiVersion: kubermatic.k8c.io/v1
+kind: Seed
+metadata:
+  # The Seed *must* be named "kubermatic".
+  name: kubermatic
+  namespace: kubermatic
+spec:
+[...]
+  etcdBackupRestore:
+    defaultDestination: minio
+    destinations:
+      minio:
+        bucketName: kkpbackup
```

Comment:
We can mention that if you used different bucket name previously, make sure to update it here.